packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -662,7 +649,7 @@ export class SnapController extends BaseController<
     state,
     environmentEndowmentPermissions = [],
     idleTimeCheckInterval = inMilliseconds(5, Duration.Second),
-    checkBlockList,
+    registry = new JsonSnapRegistry({}),
```

Comment:
Is the empty object necessary?
```suggestion
    registry = new JsonSnapRegistry(),
```

Reply:
With the current constructor yeah, not sure if there is a better pattern for having a constructor with an options bag with default options

Reply:
Thoughts?